stable/kube-slack/templates/configmap.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "kube-slack.fullname" . }}
+  labels:
+    app: {{ template "kube-slack.name" . }}
+    chart: {{ template "kube-slack.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
```

Comment:
would it be better / more flexible and future-proof to just copy a block of values into the configmap? So you don't have to copy and predefine one and one value?

Reply:
@skomma ?

Reply:
Sorry, I'll make the PR again, including the requested changes!!